apps/site/assets/ts/helpers/service.ts

```diff
@@ -91,11 +101,16 @@ export const isInFutureRating = (
   service: Service,
   currentDate: Date = new Date()
 ): boolean => {
-  if (hasNoRating(service)) {
-    return false;
+  if (hasIncompleteRating(service)) {
+    // check for rating start date
+    return service.rating_start_date
+      ? currentDate <= stringToDateObject(service.rating_start_date!)
+      : false;
   }
```

Comment:
This change lets us list a future service with a start date, for CR or Ferry services that might have a rating start date but no rating end date